dramatiq/middleware/__init__.py

```diff
@@ -38,11 +40,18 @@
     # Middlewares
     "AgeLimit", "AsyncIO", "Callbacks", "CurrentMessage", "GroupCallbacks",
     "Pipelines", "Retries", "Shutdown", "ShutdownNotifications", "TimeLimit",
-    "TimeLimitExceeded", "Prometheus",
+    "TimeLimitExceeded",
 ]
 
 
 #: The list of middleware that are enabled by default.
 default_middleware = [
-    Prometheus, AgeLimit, TimeLimit, ShutdownNotifications, Callbacks, Pipelines, Retries
+    AgeLimit, TimeLimit, ShutdownNotifications, Callbacks, Pipelines, Retries
 ]
+
+try:
+    if importlib.util.find_spec("prometheus_client"):
+        default_middleware.insert(0, Prometheus)
```

Comment:
My suggestion: if we are going to make a breaking change release removing Prometheus from the default middleware, we should also remove this line. In case there is someone who wants to install this library, but doesn't want the middleware.

i.e. the `default_middleware` list shouldn't change depending on what libraries you have installed.

Reply:
@Bogdanp, do you have an opinion about this?